dark/assets/css/styles.css

```
@@ -1126,4 +1131,4 @@ html {
   border-radius: 2rem;
   font-size: 1.1rem;
   transition: 0.2 ease;
-}
+}
```

Add the end of the line @Rishav1707 
```suggestion
}

```

---

Do this change @Rishav1707 
You still didn't done it, don't resolve the conversations again

```suggestion
}

```